src/app-host/app-host.js

```diff
@@ -1,11 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-
+/* global io:false */
 var livereload = require('./live-reload-client');
 var Messages = require('messages');
 var telemetry = require('telemetry-helper');
 
 var cordova;
-var oldExec;
 var socket = io();
```

Comment:
but we don't ignore the rule, we just tell that there's a global `io` which is not writeable (see line 2). the code does not need to become more verbose just because it's being linted.